python/ray/autoscaler/_private/kuberay/node_provider.py

```diff
@@ -446,6 +468,10 @@ def _scale_request_to_patch_payload(
         for node_type, target_replicas in scale_request.desired_num_workers.items():
             group_index = _worker_group_index(raycluster, node_type)
             group_max_replicas = _worker_group_max_replicas(raycluster, group_index)
+            group_num_of_hosts = _worker_group_num_of_hosts(raycluster, group_index)
```

Comment:
Why does scale-up work without this change?

Reply:
Oh I think the node provider actually accounts for this [here](https://github.com/ray-project/kuberay/blob/40a946a57f9aae943d7deae13a54bb1ec6288621/ray-operator/controllers/ray/raycluster_controller.go#L758), since it determines the number of pods to create using `workerReplicas * worker.NumOfHosts`. I'll remove these lines from the PR.